Jetcaster/app/src/main/java/com/example/jetcaster/util/Insets.kt

```diff
@@ -438,9 +438,9 @@ fun Insets.toInnerPadding(
     top: Boolean = true,
     end: Boolean = true,
     bottom: Boolean = true
-): InnerPadding = with(DensityAmbient.current) {
+): PaddingValues = with(DensityAmbient.current) {
```

Comment:
I also renamed this `toPaddingValues` and updated the comments in Jetsnack/Owl. Oh and added the default to the ambient per #141 . Would be good to align (and update the goo.gle/compose-insets gist).

Reply:
Yeah agreed. I'll update https://goo.gle/compose-insets next week, and then make sure everything is consistent